README.md

```diff
@@ -1,215 +1,151 @@
-# SQLiteToExcel
-This is a Light weight Library to Convert SQLite Database to Excel and Convert Excel to SQLite.
+# SQLite2XL
+A HMOS library to convert SQLite Database to Excel and Convert Excel to SQLite.
 
-## Featured In
-[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-SQLite%20To%20Excel-brightgreen.svg?style=flat)](https://android-arsenal.com/details/1/5221)
-<a href="http://www.methodscount.com/?lib=com.ajts.androidmads.SQLite2Excel%3Alibrary%3A1.0.2"><img src="https://img.shields.io/badge/Methods and size-core: 130 | deps: 22949 | 16 KB-e91e63.svg"/></a>
-## Sample App
-The sample app in the repository is available on Google Play:
-
-<a href='https://play.google.com/store/apps/details?id=com.ajts.androidmads.sqlite2xlDemo&utm_source=AndroidMads&utm_campaign=AndroidMads&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' width='150px' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png'/></a>
+# Source
+Inspired by [androidmads/SQLite2XL](https://github.com/androidmads/SQLite2XL) - version v1.0.2
 
 ## Features
-### 1.0.3
-Added Pull Request from https://github.com/cleathley
-1. Added the ability to provide a custom formatter  to the export.
-2. Added the ability to exclude columns from the export.
-3. Added `pretty` name mapping to export
-### 1.0.2
-1. Added support to add new column from excel while importing, if the column is not exists.
-### 1.0.1 
-1. Added Functionality to Import Excel into SQLite Database.
-2. Added Functionality to Export Blob into Image.
-3. Added Functionality to Export List of tables specified.
-## How to Download
-add the following library in your app level gradle file
+1) This library provides user the functions to export SQLite Database to Excel and import Excel data to SQLite database.
```

Comment:
The features must be with respect to the library (not sample application). Please check and correct this.

Reply:
Done